cves/CVE-2015-8218.yml

```diff
@@ -355,7 +386,7 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 129
```

Comment:
129 vs. 119?

Reply:
The CVE specifically mentioned OoB array access, rather than OoB memory access, so I went with 129. I've added a note about this